### PR TITLE
Hook gossip to Drand client and Drand command-line client

### PR DESCRIPTION
**Pull request by petar**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,19 +2,26 @@ module github.com/drand/drand
 
 go 1.14
 
+replace github.com/drand/drand/cmd/relay-gossip => /Users/petar/src/github.com/drand/drand/cmd/relay-gossip
```
Comment (willscott) [inline go.mod:5]: nit: remove

**Comment by willscott** [comment]
> can you make a `go.mod` in the `cmd/client` directory, in the same way as the one already in `cmd/relay-gossip`?

Actually, that seems like a not-very-natural package boundary.
Instead, can you remove the go.mod / go.sum from the `cmd/relay-gossip`? they're no longer needed if all the libp2p deps have made their way into the main `go.mod`.

**Comment by petar** [comment]
@nikkolasg This PR is failing a test in cmd/drand-cli called TestStartWithoutGroup, even though the PR does not touch any code related to (used by) this test. Can you take a look?

This test fails for me on master as well, when I run it locally.

**Comment by willscott** [comment]
fix for that in https://github.com/drand/drand/pull/420

**Comment by petar** [comment]
@willscott Why is it not letting me merge this PR with your last commit on it? It says you approved it.

**Pull request merged**